webprojects/amo.md

```diff
@@ -0,0 +1,93 @@
+# AMO: how to localize and test
+
+## Product at a glance
+
+AMO refers to https://addons.mozilla.org/. It is one of Mozilla’s biggest web properties, with 12 million users per month. The product consists three key components that require localization:
+
+* [AMO server](https://pontoon.mozilla.org/projects/amo) is the backend component of AMO
+* [AMO frontend](https://pontoon.mozilla.org/projects/amo-frontend/) covers the top page of `about:addons` and the user-facing pages of addons.mozilla.org on Firefox for Android.
+* [Disvoery Pane](https://pontoon.mozilla.org/projects/amo-frontend/) is home of curated addons available on `about:addons`.
+
+Product updates are deployed every Thursday to production. Localizable strings can be extracted on Tuesdays, and changes landed by 5pm UTC each Tuesday go live on Thursday. To report issues, go to: https://github.com/mozilla/addons-server/issues/new.
```

Comment:
Don't write URLs unless you want people to see the URL (e.g. when showing demo servers), create proper links.

`To report problem you can use [GitHub issues](https://github.com/mozilla/addons-server/issues/new)`.